esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
Agreed.

We should not be optimizing for column counts in the data itself. If we want columns to be minimized, let's write build/display logic to handle that - the data should always be as granular as possible.